README.md

```diff
@@ -188,6 +189,42 @@ $ openapi-transformer-toolkit oas2tson -i ./openapi.yml -o ./schemas -p paths
 -h, --help                 Display help for command
 ```
 
+#### Notes
```

Comment:
Who is the audience for these notes? The users or who develops this module?
In the first scenario, we can leave it here. Otherwise, we should move it to something like `CONTRIBUTING.md`

Reply:
People consuming the output of `openapi-transformer-toolkit`. Reasoning:

- `securitySchemas` fails -- `-p components.securitySchemas` failing is not a bug (and why)
- what to use for `-p` -- this question prompted the issue that led to this PR
- prefixing -- so people understand why this choice was made and what prefixes to expect
- TSON paths don't work for Fastify -- the original intent of `oas2tson` was to support Fastify with the JSON schema type provider; I was hoping paths would work, but examining the output realized they wouldn't; preparing others for a similar disappointment
- change to `nullable` -- this difference was a challenge for me getting the test spec right and required some digging to find the answer